torch/testing/_internal/common_methods_invocations.py

```diff
@@ -21078,6 +21078,12 @@ def __init__(
                          dtypes=(torch.float32,), device_type='cpu'),
         ),
     ),
+    PythonRefInfo(
+        "_refs.nn.functional.glu",
+        torch_opinfo_name="nn.functional.glu",
+        supports_nvfuser=False,
```

Comment:
cc @IvanYashchuk, this seems like an easy pointwise operation, I'm surprised nvfuser doesn't support it

Reply:
`torch.tensor_split` is not supported yet.
The error is `Node slice_in_dim_default with target prims.slice_in_dim.default does not support nvfuser`.
`prims.slice_in_dim` is used in the `torch._refs.tensor_split` implementation.